lib/premailer/rails/css_loaders/asset_pipeline_loader.rb

```diff
@@ -14,7 +14,7 @@ def load(url)
 
         def file_name(url)
           prefix = [
-            ::Rails.configuration.relative_url_root,
+            ::Rails.configuration.relative_url_root&.chomp('/'),
             ::Rails.configuration.assets.prefix,
             '/'
           ].join
```

Comment:
It probably would make more sense to delegate this to `File.join`, which takes care of removing redundant `/`:

```suggestion
          prefix = File.join(
            ::Rails.configuration.relative_url_root,
            ::Rails.configuration.assets.prefix,
            '/'
          )
```

Reply:
Yes, thank you! 🙌